AirCasting/Utils/Keychain/KeychainManager.swift

```diff
@@ -0,0 +1,51 @@
+// Created by Lunar on 16/08/2022.
+//
+
+import Foundation
+
+class KeychainManager {
```

Comment:
I would much rather see that as a protocol-implementation pair + Resolver.

It will make testing code that uses keychain much easier